lib/sles4sap/console_redirection.pm

```diff
@@ -43,33 +43,133 @@ sub handle_login_prompt {
     my $pwd = get_var('_SECRET_SUT_PASSWORD', $testapi::password);
     set_serial_term_prompt();
     # look for either password prompt or command prompt to appear
-    my $serial_response = wait_serial(qr/Password:\s*$|:~/, timeout => 20);
+    my $serial_response = wait_serial(qr/Password:\s*$|:~/, timeout => 20, quiet => 1);
 
     die 'Neither password not command prompt appeared.' unless $serial_response;
     # Handle password prompt if it appears
     if (grep /Password:\s*$/, $serial_response) {
         type_password $pwd;
         send_key 'ret';
         # wait for command prompt to be ready
-        die 'Command prompt did not appear within timeout' unless wait_serial(qr/:~|#|>/, timeout => 20);
+        die 'Command prompt did not appear within timeout' unless wait_serial(qr/:~|#|>/, timeout => 20, quiet => 1);
     }
     set_serial_term_prompt();    # set correct serial prompt
 }
 
 =head2 redirection_init
 
-    redirection_init();
+    redirection_init( [, ssh_user=>$ssh_user, target_ip=>$target_ip, ssh_tunnel_port=>$ssh_tunnel_port]);
+
+B<ssh_user>: Target login user - default value is defined by OpenQA parameter REDIRECT_TARGET_USER
```

Comment:
What is `Target` ?

Reply:
Changed naming to 'destination'... including other variables.